lib/scripts.js

```diff
@@ -230,11 +238,57 @@ var scripts = {
         }
       });
   },
-  moveToCompleted: function(job, removeOnComplete){
+  _moveToCompleted: function(job, removeOnComplete){
```

Comment:
Yes. I have understood that there is no point in trying to have a generic move function, so I will have specific methods instead.